frontend/src/components/TimeEntry/AddEntryForm.js

```diff
@@ -54,19 +55,19 @@ const additionalValues = (timeUnit, defaultDate, allValues) => {
     case 'DAY': {
       return {
         'startDateTime': allValues.startDateTime || start, //9 am by default
-        'numberHours': 8
+        'numberOfHours': 8
```

Comment:
you have environment variable

Reply:
We prefer to kee 4 and 8 by design in the UI